rskj-core/src/test/java/co/rsk/peg/RegisterBtcTransactionIT.java

```diff
@@ -147,6 +147,20 @@ void registerBtcTransaction_forARepeatedLegacyBtcTransaction_shouldNotPerformAny
         assertEquals(expectedReceiverBalance, repository.getBalance(rskReceiver));
     }
 
+    @Test
+    void registerBtcTransaction_forALegacyBtcTransactionWithNegativeHeight_shouldNotPerformAnyChange() throws Exception {
+        // Arrange
+        co.rsk.core.Coin expectedReceiverBalance = repository.getBalance(rskReceiver);
+        List<UTXO> expectedFederationUTXOs = federationSupport.getActiveFederationBtcUTXOs();
+
+        // Act
+        bridgeSupport.registerBtcTransaction(rskTx, bitcoinTransaction.bitcoinSerialize(), -1, pmtWithTransactions.bitcoinSerialize());
```

Comment:
Based on the logic internal logic, this should throw a `new RegisterBtcTransactionException("Could not validate transaction")`.

Reply:
Interesting, this test should be failing then. Maybe we are missing something here?

Reply:
Interesting, this test should be failing then. Maybe we are missing something here?

Reply:
No exception is thrown because of negative height or repeated btc transactions. The exception is caught within the BridgeSupport::registerBtcTransaction method and it prints a log. I'll add this as a future issue to review.